__tests__/info-contact-not-present-warning/.redocly.yaml

```diff
@@ -0,0 +1,42 @@
+apiDefinitions:
+  main: ./openapi.yaml
+
+lint:
+  rules: 
+    info-description: off
+    info-contact: warn
+    info-license: off
+    info-license-url: off
+
+    tag-description: off
+    tags-alphabetical: off
+
+    no-server-example.com: off
+    no-server-trailing-slash: off
+    no-empty-servers: off
+
+    parameter-description: off
+    no-path-trailing-slash: off
+    path-declaration-must-exist: off
+    path-not-include-query: off
+    path-parameters-defined: off
+    operation-description: off
+    operation-2xx-response: off
+    operation-operationId-unique: off
+
+    operation-parameters-unique: off
+    operation-tag-defined: off
+    operation-security-defined: off
+
+    operation-singular-tag: off
+
+    no-example-value-and-externalValue: off
+
+    no-unused-components: off
+    no-unresolved-refs: off
+    no-enum-type-mismatch: off
+
+    boolean-parameter-prefixes: off
+    paths-kebab-case: off
+    spec: off
```

Comment:
Can we remove those `off` everywhere where we have `extends: []`

Reply:
Good point. Thanks. Fixed.